ironic_prometheus_exporter/messaging.py

```diff
@@ -0,0 +1,30 @@
+import os
+import json
+
+from oslo_config import cfg
+from oslo_messaging.notify import notifier
+
+prometheus_opts = [
+    cfg.StrOpt('file_path', required=True,
+               help='Path for the json file where the metrics will be stored.')
+]
+
+
+def register_opts(conf):
+    conf.register_opts(prometheus_opts, group='oslo_messaging_notifications')
+
+
+class PrometheusFileDriver(notifier.Driver):
+    """Publish notifications into a File to be used by Prometheus"""
+
+    def __init__(self, conf, topics, transport):
+        self.file_path = conf.oslo_messaging_notifications.file_path
+        if not self.file_path.endswith('.json'):
+            raise Exception('The file should end with .json')
```

Comment:
I'd drop this check, why not call the file the way users want?

Reply:
Yeah make sense, since we will process the message and write in the prometheus format

Reply:
Once we have the data, @iurygregory can confirm that he is getting a node name or uuid across and we can change the path so we just overwrite data  as we go for the existing file. I think we will get it in one huge latest chunk per node so we should always be safe to replace the prior node's data.

Reply:
Also do we want a default path ? When I was talking with Mike for the test plan he asked if would be really necessary to setup configuration for the driver or if we have default values.

Reply:
I'm sure we're going to need to allow it to be configurable, but I don't know about the default path. I think we can change that if it makes sense to do so later... Maybe we even dump that into the shared folder, but that means ironic configuration that is static once we begin integrating... since we will also need to turn on metrics collection.